crates/starknet-os/src/hints/kzg.rs

```diff
@@ -138,19 +133,28 @@ fn fft(coeffs: &[BigInt], generator: &BigInt, prime: &BigInt, bit_reversed: bool
 }
 
 fn split_commitment(num: BigInt) -> (BigInt, BigInt) {
-    let low_part = &num % (BigInt::one() << COMMITMENT_LOW_BIT_LENGTH);
-    let high_part = &num >> COMMITMENT_LOW_BIT_LENGTH;
-    (low_part, high_part)
+    // Ensure the input is 384 bits (48 bytes)
+    let num = num & &((BigInt::from(1) << 384) - 1);
```

Comment:
Is this number is constant, I think that it should be defined as const to avoid/reduce magic numbers